css/layout.css

```diff
@@ -0,0 +1,66 @@
+.container {
+  margin: 0 auto;
+  padding: 0 1rem;
+}
+
+.row {
+  display: flex;
+}
+
+.justify-center {
+  justify-content: center;
+}
+
+.flex-column {
+  flex-direction: column;
+}
+
+.column-full {
+  flex-basis: 100%;
+}
+
+.column-fourth {
+  flex-basis: 25%;
+}
+
+.column-half {
+  flex-basis: 50%;
+}
+
+.align-center {
+  align-items: center;
+}
+
+.space-between {
+  justify-content: space-between;
+}
+
+.flex-row {
+  display: flex;
+}
+
+@media screen and (width <= 375px) {
+  .row {
+    flex-direction: column;
+  }
+
+  .column-fourth {
+    flex-basis: 100%;
+  }
+}
+
+@media screen and (width >= 768px) {
+  .row {
+    flex-wrap: wrap;
+  }
```

Comment:
Similar to above comment, why not just let your item wrap by default? Make a separate utility class for it but that way it'll force additional items to go onto new lines.